lib/hashes.md

```diff
@@ -5,40 +5,42 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A hash is the python form of a dictionary.  They are similar to arrays, but whereas an array uses an integer to index and organize the objects located within it, whereas a hash is more variable.  It can use any value to help organize
+the list.  Hashes can help group different items/objects in the list into associations.  Like what I know from python, the key is the sorting, association-making variables that help group, label, and identify the values assigned to them.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+Since hashes do not store the items in a numbered list, so if you need the data stored in a specific order, so that it can be called by referencing a specific number in that sequence, then hashes can be useful.  They can group and organize data/objects,
```

Comment:
> Since hashes do not store the items in a numbered list, so if you need the data stored in a specific order, so that it can be called by referencing a specific number in that sequence, then hashes can be useful. 

It seems from what you are saying that Hashes would _not_ be useful in this instance.